app/src/main/java/com/talent/animescrap/ui/activities/PlayerActivity.kt

```diff
@@ -15,6 +16,7 @@ import android.os.Build
 import android.os.Bundle
 import android.os.CountDownTimer
 import android.support.v4.media.session.MediaSessionCompat
+import android.util.Log
```

Comment:
Imports can be optimized

Reply:
Ahh!! Will take care next time.

Reply:
Somehow this change is breaking release build. I wonder why

Reply:
What the logs are saying? Anyways I'll try to look into it.

Reply:
Nvm, Fixed it. 0e900b6